e2e/pages/adf/dataTablePage.ts

```diff
@@ -104,6 +192,11 @@ export class DataTablePage {
         selectMode.click();
     }
 
+    checkRowIsSelectedByName(content) {
```

Comment:
We should have only one method for checkRowIsSelected and one for CheckRowIsNotSelected. Shouldn't matter which column/value we are using to identify the row we needed: currently i saw we have checkRowIsSelected , checkRowIsSelectedByName, checkRowIsSelectedByNumber. Please keep only one, rename it with a general name and remove the others